test/e2e/scheduling/taint_based_evictions.go

```diff
@@ -1,195 +0,0 @@
-/*
-Copyright 2018 The Kubernetes Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package scheduling
-
-import (
-	"errors"
-	"fmt"
-	"time"
-
-	"k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/fields"
-	clientset "k8s.io/client-go/kubernetes"
-	schedulerapi "k8s.io/kubernetes/pkg/scheduler/api"
-	"k8s.io/kubernetes/test/e2e/framework"
-	e2elog "k8s.io/kubernetes/test/e2e/framework/log"
-	e2enode "k8s.io/kubernetes/test/e2e/framework/node"
-	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
-
-	"github.com/onsi/ginkgo"
-)
-
-func newUnreachableNoExecuteTaint() *v1.Taint {
-	return &v1.Taint{
-		Key:    schedulerapi.TaintNodeUnreachable,
-		Effect: v1.TaintEffectNoExecute,
-	}
-}
-
-func getTolerationSeconds(tolerations []v1.Toleration) (int64, error) {
-	for _, t := range tolerations {
-		if t.Key == schedulerapi.TaintNodeUnreachable && t.Effect == v1.TaintEffectNoExecute && t.Operator == v1.TolerationOpExists {
-			return *t.TolerationSeconds, nil
-		}
-	}
-	return 0, errors.New("cannot find toleration")
-}
-
-var _ = SIGDescribe("TaintBasedEvictions [Serial]", func() {
```

Comment:
Does this test any scheduler functionality? I think this should become a node test first and let SIG node decide whether an integration test make sense.

Reply:
Well, I think taintBasedEvictions have historically  been owned by sig-scheduling, even though the component which is responsible for this feature in nodelifecycle manager and the reason being we apply `NoExecute` taint and taints have been owned by sig-scheduling. However, you're right that it doesn't test scheduler functionality.

Reply:
At some point it was, but I remember that @davidopp mentioned that it is now owned by SIG Node, which makes sense.